frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -219,6 +226,15 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
                   )}
                 </StyledClearIcon>
               </StyledHoverOptions>
+              <StyledFlexEnd hidden={key !== keyWithQLinksOpen}>
+                {state?.projectData?.key?.linkGroups && (
+                  <ProjectLinks
+                    linkGroups={state?.projectData?.key?.linkGroups}
```

Comment:
this needs to reference the variable
```suggestion
                    linkGroups={state?.projectData?.[key]?.linkGroups}
```

edit: in two other places here as well